share/messages_to_clients.txt

```diff
@@ -9382,16 +9382,12 @@ ER_EXCLUDE_ENCRYPTION_THREADS_RUNNING
 ER_EXPLICIT_DECRYPTION_OF_ONLINE_ENCRYPTED_TABLESPACE
   eng "Tablespace %s is encrypted by encryption threads and cannot be explicitly decrypted. Please use default-table-encryption=ONLINE_FROM_KEYRING_TO_UNENECRYPTED to decypt the tablespace"
 
-<<<<<<< HEAD
 ER_PXC_GR_INIT
   eng "%s"
 
-||||||| merged common ancestors
-=======
 ER_SEQUENCE_TABLE_SIZE_LIMIT
```

Comment:
I think we should change the order here. PXC specific error messages should come after PS error messages, to ensure that for PS error messages, the error codes are the same in PS and PXC.

Reply:
fixed